### PR TITLE
update about screen layout

### PR DESCRIPTION
**Pull request by remy2495**
Fix the layout

### CODE REVIEW

File: pages/about.jsx [hunk 1]
```diff
@@ -2,25 +2,54 @@ import Link from "next/link";
 
 const About = () => {
   return (
-    <main>
-      <section id="navigation_pane">
-        <div id="menu-button">
+    <main
+      style={{
```
Comment (alserembani94) [inline pages/about.jsx:7]: Please add ID to this component for verbosity.

File: pages/about.jsx [hunk 2]
```diff
@@ -2,25 +2,55 @@ import Link from "next/link";
 
 const About = () => {
   return (
-    <main>
-      <section id="navigation_pane">
-        <div id="menu-button">
+    <main
+      id="about-screen"
```
Comment (alserembani94) [inline pages/about.jsx:6]: I don't like this ID.

Comment (remy2495) [reply pages/about.jsx:6]: I don't think this is a problem.

**Review by alserembani94 — APPROVED** [review]
LGTM (Looks good to me)

**Pull request merged**